runtime/src/lib.rs

```diff
@@ -213,7 +211,7 @@ impl frame_system::Config for Runtime {
     /// Portion of the block weight that is available to all normal transactions.
```

Comment:
do we need all the comments?
maybe make it similar to the standalone chain's runtime/src/lib.rs ?

Reply:
@ltfschoen Please create new ticket for it.